contracts/GoodGhosting.sol

```diff
@@ -106,8 +106,9 @@ contract GoodGhosting is Ownable, Pausable {
         uint256 _segmentPayment,
         uint256 _earlyWithdrawalFee,
         uint256 _customFee,
-        address _dataProvider
-    ) public {
+        address _dataProvider,
```

Comment:
Do we use this data provider param for anything?

Reply:
using it here @fodisi https://github.com/Good-Ghosting/goodghosting-smart-contracts/blob/feature/whitelist-players/contracts/GoodGhosting.sol#L132

Reply:
Oh, I see. Sorry. I was reviewing this on my cellphone and it looked like it was a new parameter. than I checked for any new assignment to it and didn't see it, so I posted the question above. Totally my bad - sorry about it.